cmd/tests/cmd_run_test.go

```diff
@@ -1837,3 +1837,47 @@ func TestUIRenderOutput(t *testing.T) {
 		})
 	}
 }
+
+// TestRunStaticArchives tests that the static archives are working as expected.
+// each archive contains the following files/catalogs:
+// ├── a.js
+// ├── foo
+// │   └── bar.js
+// ├── sample
+// │   └── data.json
+// └── script.js
+// archive was made using binary & platform from the test name
+func TestRunStaticArchives(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		archive string
+	}{
+		{archive: "archive_v0.42.0_linux.tar"},
+		{archive: "archive_v0.42.0_windows.tar"},
```

Comment:
Ah, I was picturing more of a somewhat complex CI workflow where we actually run `k6 archive` on one OS, and then run the test on the other. That way we wouldn't need to commit the packages. But I guess this is fine as well.

Reply:
>I was picturing more of a somewhat complex CI workflow

In my mind, I have the same picture, but that picture also sounds costly for the scope of this PR, so this test is a tread-off. Maybe in the future, we could improve it  (should we create the task then :thinking: ).

Reply:
Also, this has the benefit of being able to test archives generated from old k6 version on new k6 versions, which we won't easily be able to do with `k6 archive`. So, both are good to have eventually, but this is the more important one and the one easier to implement.